src/TwigComponent/tests/Fixtures/templates/components/Button.html.twig

```diff
@@ -0,0 +1,5 @@
+{% props label, primary = true %}
+
+<button class="{{ primary ? 'primary' : 'secondary' }}">
```

Comment:
We should use `attributes` here:

```twig
<button {{ attributes.defaults({ class: primary ? 'primary' : 'secondary' }) }}>
```

The reason is that, we need to test that anything that is a prop is NOT added as an attribute (and I think this may not be implemented yet?). So given this example:

```html
<twig:Button label='Click me' :primary='false'/>
```

We need to assert that, even if we use `{{ attributes }}`, both `label` and `primary` don't show up as attributes.

Reply:
Hum, I am not sure we should have a difference between argument and prop. Why do you think we should differentiate theses two notions?